autospec/tarball.py

```diff
@@ -232,6 +232,8 @@ def name_and_version(name_arg, version_arg, filemanager):
     pattern_options = [
         r"(.*?)[\-_](v*[0-9]+[a-zalpha\+_spbfourcesigedsvstableP0-9\.\-\~]*)\.orig\.tar",
         r"(.*?)[\-_](v*[0-9]+[alpha\+_spbfourcesigedsvstableP0-9\.\-\~]*)\.src\.(tgz|tar|zip)",
+        # handle font packages with names ending in -nnndpi
+        r"(.*-[0-9]+dpi?)[\-_](v*[0-9]+[alpha\+_sbpfourcesigedsvstableP0-9\.\-\~]*)\.(tgz|tar|zip)",
```

Comment:
The `dpi?` bit means to match `d`, `p`, and an optional `i` character. That's probably not the intention...

Reply:
correct...

Reply:
I removed the '?'